### ISSUE TITLE
train_fm.py: FwdCNN' object has no attribute 'intype'

### ISSUE THREAD

**Issue by adegirmenci** [Atcold/pytorch-PPUU#20]
```
python train_fm.py -model_dir ~/PPUU-models

[loading data shard: traffic-data/state-action-cost/data_i80_v0/trajectories-0400-0415/all_data.pth]
[loading data shard: traffic-data/state-action-cost/data_i80_v0/trajectories-0500-0515/all_data.pth]
[loading data shard: traffic-data/state-action-cost/data_i80_v0/trajectories-0515-0530/all_data.pth]
Number of episodes: 5596
[loading data splits: traffic-data/state-action-cost/data_i80_v0/splits.pth]
[loading data stats: traffic-data/state-action-cost/data_i80_v0/data_stats.pth]
[loading car sizes: traffic-data/state-action-cost/data_i80_v0/car_sizes.pth]

[will save model as: ~/PPUU-models/model=fwd-cnn-layers=3-bsize=8-ncond=20-npred=20-lrt=0.0001-nfeature=256-dropout=0.0-gclip=5.0-warmstart=0-seed=1]
Traceback (most recent call last):
  File "train_fm.py", line 107, in <module>
    model.intype('gpu')
  File "~/miniconda3/envs/PPUU/lib/python3.7/site-packages/torch/nn/modules/module.py", line 535, in __getattr__
    type(self).__name__, name))
AttributeError: 'FwdCNN' object has no attribute 'intype'```

**Comment by mbhenaff** [comment]
This should be fixed now: 1d2c6e0.

**Comment by Atcold** [comment]
I've removed `intype` from `train_fm.py`.
